scripts/Tools/acme_bisect

```diff
@@ -76,8 +76,8 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
 ###############################################################################
 def acme_bisect(testname, good, bad, testroot, compiler, project, baseline_name, check_namelists, check_throughput, check_memory):
 ###############################################################################
-    create_test = os.path.join(CIME.utils.get_acme_scripts_root(), "create_test")
```

Comment:
Awesome. Before I forget to check, was get_acme_scripts_root deleted from CIME.utils?